File: aqi-estimator.server/src/main/java/com/codefu/refactor/api/aqiestimatorserver/DTO/clientResponses/AQIPredictionResponse.java
package com.codefu.refactor.api.aqiestimatorserver.DTO.clientResponses;

import com.codefu.refactor.api.aqiestimatorserver.enums.AQICategory;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AQIPredictionResponse {
    Integer modelPredictionValue;
    SensorCalculatedAqiAverages sensorCalculationHighestAqi;
    AQICategory aqiCategory;
}

File: aqi-estimator.client/src/app/shared/models/responses/AQIPredictionResponse.ts
import {SensorCalculatedAqiAverages} from "./SensorCalculatedAqiAverages";
import {AQICategory} from "../../enums/api-responses/AQICategory";

export interface AQIPredictionResponse {
  modelPredictionValue: number;
  sensorCalculationHighestAqi: SensorCalculatedAqiAverages;
  aqiCategory: AQICategory;
}
